primitive/bls12_381/src/coordinate.rs

```diff
@@ -86,7 +86,7 @@ const PARAM_B: Fq = Fq([
 pub struct Bls381Affine {
     x: Fq,
     y: Fq,
-    is_infinity: bool,
+    is_identity: bool,
```

Comment:
From terminology perspective, this should be `is_infinity` I think.
The statement is that the Affine point is infinity so the point is identity in group.